src/main/java/org/reaktivity/nukleus/maven/plugin/internal/ast/parse/AstParser.java

```diff
@@ -585,8 +617,155 @@ private static int parseInt(
         }
     }
 
-    private static int parseInt(String text)
+    private static int parseInt(
+        String text)
     {
         return Integer.decode(text);
     }
+
+    private static long parseLong(
+        RuleContext ctx)
+    {
+        if (ctx == null)
+        {
+            return 0;
+        }
+        else
+        {
+            return parseLong(ctx.getText());
+        }
+    }
+
+    private static long parseLong(
+        String text)
+    {
+        return Long.decode(text.substring(0, text.length() - 1));
+    }
+
+    private static String parseString(
+        RuleContext ctx)
+    {
+        return ctx != null ? ctx.getText() : null;
```

Comment:
Do we actually need to defend against `null` context here?
Would prefer to remove this if it would hide a logic error in the code that needs to be addressed either now or later.